### PR TITLE
fix(frontend): mobile design

### PR DESCRIPTION
**Pull request by ogerly**
## 🍰 Pullrequest
mobile menu has transparent background color

image

### CODE REVIEW

**Comment by ogerly** [comment]
image

**Comment by ogerly** [comment]
Bildschirmfoto vom 2023-01-16 14-14-50


image

**Comment by ogerly** [comment]
image

**Comment by ogerly** [comment]
Bildschirmfoto vom 2023-01-16 14-54-07


Bildschirmfoto vom 2023-01-16 14-53-40

**Comment by ogerly** [comment]
image

**Comment by ogerly** [comment]
image

**Comment by ogerly** [comment]
image

**Comment by ogerly** [comment]
https://user-images.githubusercontent.com/1324583/212827442-cad97d1f-3a93-4b34-8fb6-5b854f8a94c7.mp4

File: frontend/src/components/Inputs/InputHour.vue [hunk 1]
```diff
@@ -32,15 +32,15 @@ export default {
...
     },
-    name: { type: String, required: true, default: 'Time' },
-    label: { type: String, required: true, default: 'Time' },
-    placeholder: { type: String, required: true, default: 'Time' },
-    value: { type: Number, required: true, default: 0 },
-    validMaxTime: { type: Number, required: true, default: 0 },
+    name: { type: String, required: true },
+    label: { type: String, required: true },
+    placeholder: { type: String, required: true },
+    value: { type: Number, required: true },
```
Comment (Mogge) [inline frontend/src/components/Inputs/InputHour.vue:38]:
```suggestion
    value: { type: Number, required: true, default: 0 },
```

File: frontend/src/components/Inputs/InputHour.spec.js [hunk 2]
```diff
@@ -58,7 +58,7 @@ describe('InputHour', () => {
       })
 
       it('has the value 0', () => {
-        expect(wrapper.vm.currentValue).toEqual(0)
+        expect(wrapper.vm.currentValue).toEqual(null)
```
Comment (Mogge) [inline frontend/src/components/Inputs/InputHour.spec.js:61]:
```suggestion
        expect(wrapper.vm.currentValue).toEqual(0)
```

File: frontend/src/components/Inputs/InputHour.vue [hunk 3]
```diff
@@ -32,15 +32,15 @@ export default {
...
+    validMaxTime: { type: Number, required: true },
   },
   data() {
     return {
-      currentValue: 0,
+      currentValue: null,
```
Comment (Mogge) [inline frontend/src/components/Inputs/InputHour.vue:43]:
```suggestion
      currentValue: 0,
```

**Review by Mogge — DISMISSED** [review]

**Comment by ogerly** [comment]
image

**Review by ulfgebhardt — APPROVED** [review]

**Pull request merged**